app/constants/authorities.ts

```diff
@@ -3,3 +3,5 @@ export const PUBLIC_DATA_URL =
 
 export const AUTHORITIES_LIST_URL =
   'https://raw.githubusercontent.com/Path-Check/safeplaces-frontend/develop/healthcare-authorities.yaml';
+
+export const AUTHORITY_NAME = 'Boston Public Health Commission';
```

Comment:
Can we hook this up to a env variable? It would be nice to be able to update this without needing to make a merge into the codebase.

Reply:
I think that wouldn't be too hard - but yeah, wouldn't hurt to merge this so you can start using it for test builds while I add that